helm/charts/hydra/values.yaml

```diff
@@ -139,7 +139,7 @@ deployment:
       - ALL
     readOnlyRootFilesystem: true
     runAsNonRoot: true
-    runAsUser: 1000
+    runAsUser: 100
```

Comment:
What's the difference?

Reply:
In our images we have an `ory` user which has pid 100, why not use it :)

Reply:
Ah! makes sense :)